fern/ir-types-v28/definition/api.yml

```diff
@@ -0,0 +1,4 @@
+name: ir
+docs: |
```

Comment:
note: update the docs for this ir to include the breaking change for `referencedTypes`